security/keystore-explorer/Portfile

```diff
@@ -29,12 +31,12 @@ long_description        ${description} with the following features: \
 
 homepage                https://keystore-explorer.org/
 
-checksums               rmd160  b415bf88ff780e665792f90d58d54d15b713f2b8 \
-                        sha256  4fdbe6c2af4bf24b7ebab4410bcdb25c220b1094bfdc0c816d51461cd8a75077 \
-                        size    8231162
+checksums               rmd160  17cc046dbdf5646dfd280ab02decd3953398a2b8 \
+                        sha256  c539067c94c47280e666fac6255e3f15d0b00a110920ebd3e77e631b41b608fe \
+                        size    8231142
 
 java.version            1.8+
-java.fallback           openjdk8
+java.fallback           openjdk15
```

Comment:
It is explained on the ticket that this is the version upstream is using. However `openjdk15` will only be supported until 2021-04 when `openjdk16` is released, and at which point `openjdk15` should probably be marked obsolete in MacPorts. LTS Java releases `openjdk8` and `openjdk11` are supported for at least a few more years; can one of those be used instead?

Reply:
I've reverted java.fallback so we'll get support until 2026 at least.